userprefs.php

```diff
@@ -474,7 +474,7 @@ function echo_proofreading_tab($user)
         _('Interface Type'), 'i_type', 'facetype',
         $user->profile->i_type,
         'radio_group',
-        [0 => _("Standard"), 1 => _("Enhanced")]
+        [0 => _("Standard"), 1 => _("Enhanced"), 2 => _("Unified")]
```

Comment:
do we need to add code to fallback in case we switch sandboxes and our pref is set to 2?

Reply:
Actually, when I did my testing, I ran into that. I wanted to look at something in the enhanced interface, so switched to the main TEST sandbox, and lo-and-behold, there wasn't a proofing interface chosen. However, since I knew I wanted to look specifically at an existing interface, I just chose it. I tried just now and got a `unknown i_type: '2'` error.

So, good idea.